projects/core/src/lib/layers/symbology/raster-symbology-editor/raster-symbology-editor.component.html

```diff
@@ -126,15 +126,19 @@
                 </ng-template>
             </mat-expansion-panel>
 
```

Comment:
Opening it with NDVI (NDVI data from MODIS) I get get some lag as well. Not so for Land Cover, for example, which has only 16 different values. Is this the same for you? If so, then it's the creation of the 256 geoengine-color-attribute-input elements that takes a long time when opening a raster layer with a lot of different values.

It might be preferable to only create them when the color table panel gets unfolded